deploy/utils/backup_script/run_backup.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+LOCKFILE=/tmp/backup_script.lock
+
+if [ -e "$LOCKFILE" ]; then
+    echo "Script is running now. Lockfile enabled."
+    exit 1
+fi
+
+touch $LOCKFILE
+
+source /root/.bashrc
```

Comment:
is it required?

Reply:
The answer is **No**, as a rule.
Shell scripts should be self-describing and contain everything needed to get the job done.
A simple test is to run the script as `bash --norc`.
Implied options are not welcome.